jira-slack-server-integration/jira-slack-server-integration-plugin/src/test/java/com/atlassian/jira/plugins/slack/dao/impl/DefaultConfigurationDaoTest.java

```diff
@@ -123,11 +127,14 @@ public void findByProjectId() {
     @Test
     public void findByChannel() {
         ProjectConfigurationAO[] configurationsList = {projectConfigurationAO, projectConfigurationAO2};
-        when(ao.find(ProjectConfigurationAO.class, ProjectConfigurationAO.COLUMN_CHANNEL_ID + " = ?", "C"))
+        when(ao.find(eq(ProjectConfigurationAO.class), queryCaptor.capture().select()
+                .where(ProjectConfigurationAO.COLUMN_TEAM_ID + " = ?", "T")
+                .where(ProjectConfigurationAO.COLUMN_CHANNEL_ID + " = ?", "C")))
```

Comment:
The captor is used here incorrectly. Please see how it is used in `insertProjectConfiguration()` in this test class.